core/src/autogluon/core/_setup_utils.py

```diff
@@ -27,6 +27,7 @@
     "tqdm": ">=4.38,<5",  # Major version cap
     "Pillow": ">=9.3,<9.6",  # "<{N+2}" upper cap
     "torch": ">=1.13,<2.1",  # "<{N+1}" upper cap
+    "pytorch-lightning": ">=2.0.0,<2.1",  # "<{N+1}" upper cap
```

Comment:
Is it compatible with torch 1.13, which is the minimum requirement we have above

Reply:
Yes, lightning 2.0 also supports torch 1.13. See the doc: https://lightning.ai/docs/pytorch/latest/versioning.html#compatibility-matrix